src/analysis/retail/monk/mistweaver/modules/features/RisingMistBreakdown.tsx

```diff
@@ -178,33 +187,64 @@ class RisingMistBreakdown extends Analyzer {
         color: SPELL_COLORS.DANCING_MISTS,
         label: 'Dancing Mists',
         spellId: talents.DANCING_MISTS_TALENT.id,
-        value: this.risingMist.extraVivhealingFromDancingMistRems,
+        value: this.risingMist.vivhealingFromDancingMistRems,
         valuePercent: false,
       },
       {
         color: SPELL_COLORS.RAPID_DIFFUSION,
         label: 'Rapid Diffusion',
         spellId: talents.RAPID_DIFFUSION_TALENT.id,
-        value: this.risingMist.extraVivHealingFromRapidDiffusionRems,
+        value: this.risingMist.vivHealingFromRapidDiffusionRems,
         valuePercent: false,
       },
       {
         color: SPELL_COLORS.RENEWING_MIST,
         label: 'Hardcast',
         spellId: SPELLS.RENEWING_MIST_HEAL.id,
-        value: this.risingMist.extraVivHealingFromHardcastRems,
+        value: this.risingMist.vivHealingFromHardcastRems,
         valuePercent: false,
       },
     ];
     return (
       <>
-        <strong>{this.risingMist.extraVivCleaves}</strong> total extra cleaves via{' '}
-        <SpellLink spell={talents.INVIGORATING_MISTS_TALENT} />
-        <br />
-        <SpellLink spell={SPELLS.VIVIFY} /> healing via extended{' '}
-        <SpellLink spell={talents.RENEWING_MIST_TALENT} />
-        <br />
-        by source:
+        <strong>{this.risingMist.vivCleaves}</strong> total extra{' '}
+        <SpellLink spell={talents.INVIGORATING_MISTS_TALENT} /> hits from extended <br />
+        <SpellLink spell={talents.RENEWING_MIST_TALENT} /> by source:
+        <hr />
+        <DonutChart items={items} />
+      </>
+    );
+  }
+
+  zenPulseTooltip() {
+    const items = [
+      {
+        color: SPELL_COLORS.DANCING_MISTS,
+        label: 'Dancing Mists',
```

Comment:
```suggestion
        label: TALENTS_MONK.DANCING_MISTS.name,
```
not required but could make things cleaner